charts/kube-master/templates/configmap.yaml

```diff
@@ -255,3 +255,27 @@ data:
       restartPolicy: Never
       serviceAccountName: default
       terminationGracePeriodSeconds: 30
+{{- if .Values.admissionConfig.enabled }}
+  admission.yaml: |-
+    apiVersion: apiserver.config.k8s.io/v1
+    kind: AdmissionConfiguration
+    plugins:
+    - name: ValidatingAdmissionWebhook
+      configuration:
+        apiVersion: apiserver.config.k8s.io/v1
+        kind: WebhookAdmissionConfiguration
+        kubeConfigFile: "/etc/kubernetes/admission/kubeconfig"
+    - name: MutatingAdmissionWebhook
+      configuration:
+        apiVersion: apiserver.config.k8s.io/v1
+        kind: WebhookAdmissionConfiguration
+        kubeConfigFile: "/etc/kubernetes/admission/kubeconfig"
+  admission-kubeconfig: |-
+    apiVersion: v1
+    kind: Config
+    users:
+    - name: '*'
```

Comment:
I have never seen this. Does this work? 🤯

Reply:
I've seen this somewhere but can't recall where. I guess we need to try it out.

Reply:
[Here](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#authenticate-apiservers)